### DISCUSSION TITLE
release 0.9.91 is it last for MacOS 10.13. ?

### DISCUSSION THREAD

**Discussion by Vladistone** [Hammerspoon/hammerspoon#3429] [General]
I would like to be warned about the minimum requirements for the OS at least in the release description.

**Comment by cmsj** [comment]
Yes, 0.9.92 requires 10.15. The built-in update checker should be smart enough to not offer you an update that doesn't run on your version of macOS, but I agree, we should mention it in the release notes.

**Reply by Vladistone** [reply]
I meant when the app is first time loaded, and there is no update function in sight!
in such a situation, the user is forced to act like a blind kitten - by touch ... sorting through versions at random

and in this case (MacOS 10.13) no patches should be expected? Isn`t it?

**Reply by cmsj** [reply]
I've added a note on the front page of our website to direct users of older hardware to the [Release Notes](https://www.hammerspoon.org/releasenotes) page, which now includes the minimum OS version for each release, and a link to the GitHub release page where they can download that version. Hope that help!
